src/ripple/app/consensus/RCLConsensus.h

```diff
@@ -355,14 +355,15 @@ class RCLConsensus
         /** Validate the given ledger and share with peers as necessary
 
             @param ledger The ledger to validate
+            @param set The consensus transaction set
```

Comment:
I'd rename this to something that says what's inside the set. Maybe `txSet` or `txns`.

Reply:
This was a good suggestion, so I also did it in several other places that were using `set`.